login/UserResponceServices.js

```diff
@@ -0,0 +1,26 @@
+const UserResponse = require("./UserResponse.js");
+const UserModel = require("./Models/UserModel.js");
+
+module.exports.SingIn = function SingIn (loginRequest,request,response)
+{
+    UserModel.User.findOne({raw:true,where: {login: loginRequest.login,password:loginRequest.password}})
+        .then(User=>{
+            if(!User){
+                console.log("Данные введены неправильно");
+                return;
+            }
+            else {
+                let userTables = JSON.parse(JSON.stringify(User));
+                let userResponce = new UserResponse();
+                userResponce.roleId = userTables["role_id"];
+                userResponce.login = userTables["login"];
+                userResponce.id = userTables["id"];
+                response.send(userResponce);
+            }
+        }).catch(err=>response.send(err));
+};
+
+
+
+
+
```

Comment:
there should be only one empty line at the end

Reply:
okey,i already fixed it